syncano_cli/execute/utils.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import json
+
+from syncano_cli.logger import get_logger
+
+LOG = get_logger('syncano-execute')
+
+
+def print_response(response):
```

Comment:
To clarify. ScriptEndpoint will return the actual custom response as @ilu2112 mentioned. Trace for it will include the structure @opalczynski mentioned. But here I believe we are only dealing with script endpoints so it seems ok: either structure has a result or should be returned as-is.